docs/en/siem/field-ref.asciidoc

```diff
@@ -2,15 +2,17 @@
 [chapter, role="xpack"]
 = Field reference and mappings
 
-This section lists all fields required to populate {siem-soln} data.
+This section lists the fields required to populate all {siem-soln} data.
```

Comment:
I still think "required" needs clarification or removal.  As an alternative, perhaps something like:  "This section lists all fields used by the current version of the SIEM app when it queries, aggregates, filters, displays, or renders event data."

Reply:
@MikePaquette 
@XavierM and I tried to only list fields that are relevant for ingesting data, not internal operations. What do you think about removing any fields that are not required for ingesting data? I think this section should just list all fields required to fully populate SIEM. That way, it is a complete reference guide. What do you think?